carmajava/launch/guidance.launch

```diff
@@ -95,9 +97,10 @@
 
   <remap from="/vehicle_status" to="$(optenv CARMA_INTR_NS)/vehicle_status"/>
   <remap from="/current_pose" to="$(optenv CARMA_LOCZ_NS)/current_pose"/>  
+  
   <!-- MPC Follower Wrapper -->
   <include file="$(find mpc_follower_wrapper)/launch/mpc_follower_wrapper.launch" />
```

Comment:
Your call here should pass in the vehicle_calibration_dir argument to this launch file